servicetalk-loadbalancer/src/test/java/io/servicetalk/loadbalancer/LoadBalancerTest.java

```diff
@@ -701,7 +705,11 @@ void resubscribeToEventsWhenAllHostsAreUnhealthy() throws Exception {
 
         // Events for the new Subscriber change the state
         sendServiceDiscoveryEvents(upEvent("address-2"), upEvent("address-3"), upEvent("address-4"));
-        assertAddresses(lb.usedAddresses(), "address-2", "address-3", "address-4");
```

Comment:
In theory this will affect all uses of `assertAddresses`: can we fix `assertAddresses` directly?